x-pack/plugins/upgrade_assistant/public/application/lib/api.ts

```diff
@@ -58,6 +62,22 @@ export class ApiService {
     });
   }
 
+  public useLoadSystemIndicesUpgradeStatus() {
+    return this.useRequest<SystemIndicesUpgradeStatus>({
```

Comment:
Just a note to self that this PR is in a race with https://github.com/elastic/kibana/pull/112907. Depending on whichever is merged first, the other will need to add `responseInterceptors` to these requests.